website/pages/docs/job-specification/service.mdx

```diff
@@ -705,7 +705,7 @@ advertise and check directly since Nomad isn't managing any port assignments.
 </sup>
 <small>
   {' '}
-  Script checks are not supported for the [qemu driver][qemu] since the Nomad
+  Script checks are not supported for the [qemu driver](https://www.nomadproject.io/docs/drivers/qemu) since the Nomad
```

Comment:
We don't use the link literal, but references the links at the bottom of the page:
```suggestion
  Script checks are not supported for the [qemu] driver since the Nomad
```